testapp/proguard-rules.pro

```diff
@@ -19,3 +19,5 @@
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
 #-renamesourcefileattribute SourceFile
+-keepnames class androidx.navigation.fragment.NavHostFragment
+-keepnames class com.rakuten.tech.mobile.miniapp.MiniAppInfo { *; }
```

Comment:
Keeping the `MiniAppInfo` name seems like a security threat since it's carrying the quite sensitive information.

Reply:
@rleojoseph but this is in host app and any host app can do it on their side, and `MiniAppInfo` is also a public class.